java/src/jmri/jmrit/symbolicprog/PrintCvAction.java

```diff
@@ -111,9 +111,13 @@ public void actionPerformed(ActionEvent e) {
             writer.write(tableBottomRow, tableLeft, tableBottomRow, tableRight);
 
             writer.setFontStyle(1);  //set font to Bold
-            // print a simple heading
-            s = "                 Value                   Value                   Value\n";
+            // print a simple heading with I18N
+            s = String.format("%1$21s", Bundle.getMessage("Value")) + String.format("%1$24s", Bundle.getMessage("Value")) +
```

Comment:
You probably want to do this as a single format String:
```
s = String.format("%1$21s%1$24s%1$24s", Bundle.getMessage("Value"));
```
if that doesn't work, you can use:
```
s = String.format("%1$21s%2$24s%3$24s", Bundle.getMessage("Value"), Bundle.getMessage("Value"), Bundle.getMessage("Value"));
```